tests/CMakeLists.txt

```diff
@@ -12,48 +12,77 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-add_subdirectory( ArrayProxy )
-add_subdirectory( ArrayProxyNoTemporaries )
-add_subdirectory( ArrayWrapper )
-if( VULKAN_HPP_ENABLE_CPP20_MODULES )
-	add_subdirectory( Cpp20Modules )
-	if( VULKAN_HPP_ENABLE_STD_MODULE )
-		add_subdirectory( CppStdModule )
-	endif()
-endif()
-add_subdirectory( CppType )
+vulkan_hpp__setup_test( NAME ArrayProxy )
+vulkan_hpp__setup_test( NAME ArrayProxyNoTemporaries )
+vulkan_hpp__setup_test( NAME ArrayWrapper )
+vulkan_hpp__setup_test( NAME CppType )
 if( CMAKE_CXX_STANDARD GREATER_EQUAL 20 )
-	add_subdirectory( DesignatedInitializers )
+	vulkan_hpp__setup_test( NAME DesignatedInitializers )
```

Comment:
Maybe the name of this test is not as good as it should... Anyways, it also contained tests with C++ less than 20.